apps/protoc-gen-openapi/generator/openapi-v3.go

```diff
@@ -225,11 +231,25 @@ func (g *OpenAPIv3Generator) formatMessageRef(name string) string {
 }
 
 func (g *OpenAPIv3Generator) formatMessageName(message *protogen.Message) string {
+
```

Comment:
The name generation code (below) is essentially in here twice - I lazily copied-and-pasted it between this function and `fullMessageTypeName` in the gist. I think it would be good (but optional  in this PR) to refactor into a common function.

Reply:
You're are right - I guess I got lazy too as it just worked :)
I have moved the code to a function.